sdk/communication/AzureCommunicationChat/Source/Signaling/TrouterUtils.swift

```diff
@@ -0,0 +1,432 @@
+//
+//  TrouterUtils.swift
+//  AzureCommunicationSignaling
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import AzureCommunication
+import AzureCore
+import Foundation
+import TrouterClientIos
+
+func toEventPayload(request: TrouterRequest, chatEventId: ChatEventId) -> Any? {
+    do {
+        switch chatEventId {
+        case ChatEventId.chatMessageReceived:
+            return try toChatMessageReceivedEvent(request: request)
+        case ChatEventId.typingIndicatorReceived:
+            return try toTypingIndicatorReceivedEvent(request: request)
+        case ChatEventId.readReceiptReceived:
+            return try toReadReceiptReceivedEvent(request: request)
+        case ChatEventId.chatMessageEdited:
+            return try toChatMessageEditedEvent(request: request)
+        case ChatEventId.chatMessageDeleted:
+            return try toChatMessageDeletedEvent(request: request)
+        case ChatEventId.chatThreadCreated:
+            return try toChatThreadCreatedEvent(request: request)
+        case ChatEventId.chatThreadPropertiesUpdated:
+            return try toChatThreadPropertiesUpdatedEvent(request: request)
+        case ChatEventId.chatThreadDeleted:
+            return try toChatThreadDeletedEvent(request: request)
+        case ChatEventId.participantsAdded:
+            return try toParticipantsAddedEvent(request: request)
+        case ChatEventId.participantsRemoved:
+            return try toParticipantsRemovedEvent(request: request)
+        }
+    } catch {
+        return nil
+    }
```

Comment:
If you are just going to return `nil` on failure instead of throw, you can just change the `try` to `try?`